tests/mir-opt/inline/issue_106141.outer.Inline.panic-abort.diff

```diff
@@ -4,46 +4,26 @@
   fn outer() -> usize {
       let mut _0: usize;
 +     scope 1 (inlined inner) {
-+         let mut _1: &[bool; 1];
-+         let mut _2: bool;
-+         let mut _3: bool;
 +         scope 2 {
 +             debug buffer => const _;
++             let _1: usize;
 +             scope 3 {
-+                 debug index => _0;
++                 debug index => _1;
++             }
++             scope 4 (inlined index) {
 +             }
 +         }
 +     }
   
       bb0: {
 -         _0 = inner() -> [return: bb1, unwind unreachable];
 +         StorageLive(_1);
-+         StorageLive(_2);
-+         _1 = const _;
-+         _0 = index() -> [return: bb1, unwind unreachable];
++         goto -> bb1;
       }
   
       bb1: {
-+         StorageLive(_3);
-+         _2 = Lt(_0, const 1_usize);
-+         assert(move _2, "index out of bounds: the length is {} but the index is {}", const 1_usize, _0) -> [success: bb2, unwind unreachable];
-+     }
-+ 
-+     bb2: {
-+         _3 = (*_1)[_0];
```

Comment:
This test is defunct. This is the important line, _0 appearing as an index. Could you add an inline(never) to index?